lib/Support/Windows/MSFileSystem.inc.cpp

```diff
@@ -238,6 +238,7 @@ static bool is_separator(const wchar_t value) {
 
 // TODO: consider erasing this
 namespace {
+  [[maybe_unused]]
```

Comment:
I don't think this is used. The comment says we should consider erasing it. I think we should. Unlike the above that depends on a define and the platform in question

Reply:
Fixed.